scripts/jenkins-acceptance-tests.sh

```diff
@@ -37,3 +37,10 @@ pip install -r requirements/test-acceptance.txt
 
 cd test/acceptance
 python tests.py
+
+# Unset SELENIUM_HOST so that bok-choy doesn't try to use saucelabs
+unset SELENIUM_HOST
+# AutoAuthPage times out in PhantomJS when using https, switch to use http
```

Comment:
@benpatterson @jzoldak  I've update this and got a build working against the existing sandbox.  It looks like phantomjs handles ssl a bit differently than firefox, but I'm not sure how we have it set up.  (See http://phantomjs.org/troubleshooting.html)  For now, I've got it running the accessibility tests with http and the others with https.